package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-statement-constructor",
-  "version": "0.3.4",
+  "version": "0.3.6",
```

Comment:
Would maybe debate this is a minor update as there is new functionality added rather than just patching what is there.  But given the subscription was already subscribing that could be debated.

Happy with whichever you feel is right.

Reply:
I'd problems side tht it is a minor update.

Something that I usually dnt give much consideration - as it's the first thing to be done is bump the version - which is naive of me.

Updated